stable/nginx-ingress/values.yaml

```diff
@@ -124,14 +124,16 @@ controller:
 
   affinity: {}
     # podAntiAffinity:
-    #   requiredDuringSchedulingIgnoredDuringExecution:
-    #   - labelSelector:
-    #       matchExpressions:
-    #       - key: app
-    #         operator: In
-    #         values:
-    #         - nginx-ingress
-    #     topologyKey: "kubernetes.io/hostname"
+    #   preferredDuringSchedulingIgnoredDuringExecution:
```

Comment:
how about we include both? this way a user can decide which they prefer while also providing a quick way of setting them. I would also add a blurb on top of affinity that points to the kubernetes docs that helps users understand these options.

Reply:
Sure, done.

Reply:
This is super nit-picky so apologies... Could you format the reference to the docs like we do for the other vars
https://github.com/helm/charts/blob/master/stable/nginx-ingress/values.yaml#L136-L138

Reply:
You are right to ask for that. Done!